### ISSUE TITLE
[CLOSED] rezist on lvl 63 mob  while hit caped

### ISSUE THREAD

**Issue by netherspite** [netherspite/cc-buglist#269]
<a href="https://github.com/jovce17"><img src="https://avatars3.githubusercontent.com/u/2951412?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [jovce17](https://github.com/jovce17)**    
_Tuesday Mar 31, 2015 at 19:46 GMT_  
_Originally opened as https://github.com/ccshiro/cc-buglist/issues/269_

----

i was testing ele shaman on adds in terokar i managed to miss( rezist) on lvl 63 mob with >
6% hit chance on spells , 9 % from talents (6% from ele spec and 3 from resto spec ) 
- 3% from totem of wrath ...
  i had a total +18% hit chance ... and still manged to make a rezist hit using lightning bolt 
  http://s14.postimg.org/b125irc3l/Wo_WScrn_Shot_033115_212832.jpg

**Comment by netherspite** [comment]
<a href="https://github.com/solowt"><img src="https://avatars2.githubusercontent.com/u/10148145?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [solowt](https://github.com/solowt)**  
_Tuesday Mar 31, 2015 at 19:52 GMT_

----

my understanding is that it's impossible to have a 100% chance to hit and there's always a 1% chance to miss.

Perhaps that's what happened.  Would be nice to hear from some people who are more knowledgeable on the subject than me though.

**Comment by netherspite** [comment]
<a href="https://github.com/jovce17"><img src="https://avatars3.githubusercontent.com/u/2951412?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [jovce17](https://github.com/jovce17)**  
_Tuesday Mar 31, 2015 at 20:04 GMT_

----

actualy did some more testing  from 34 lightning bolt casts on lvl 62-63 mobs i manged to get 2 rezists ...
http://s9.postimg.org/m7r068i1b/Wo_WScrn_Shot_033115_215303.jpg
2/34 its more like a 6% not 1 % ...and its Vs lvl 62 mobs ....not 73 or BOSS

**Comment by netherspite** [comment]
<a href="https://github.com/Iliana-sc"><img src="https://avatars2.githubusercontent.com/u/1512595?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Iliana-sc](https://github.com/Iliana-sc)**  
_Tuesday Mar 31, 2015 at 20:26 GMT_

----

34 is hardly enough trials to make any kind of conclusion.

**Comment by netherspite** [comment]
<a href="https://github.com/jovce17"><img src="https://avatars3.githubusercontent.com/u/2951412?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [jovce17](https://github.com/jovce17)**  
_Tuesday Mar 31, 2015 at 20:35 GMT_

----

@Iliana-sc  if there is 1 % chance that cant be avoided via capping then true and will need more testing but ....if its not supposed to miss vs 62 mob then its a bug

**Comment by netherspite** [comment]
<a href="https://github.com/nz54445"><img src="https://avatars1.githubusercontent.com/u/8640523?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [nz54445](https://github.com/nz54445)**  
_Tuesday Mar 31, 2015 at 22:56 GMT_

----

This may be useful for the conversation:
http://www.wowwiki.com/Spell_hit

It's partly wotlk and partly tbc, but the main points are there.

Now i remember something ~ 5% resists when you are hit capped on bosses, but that's just me. I can not for the life of me find any attack tables. I am almost sure that against normal mobs you should not miss at all.

**Comment by netherspite** [comment]
<a href="https://github.com/tedrasiel"><img src="https://avatars1.githubusercontent.com/u/10154440?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [tedrasiel](https://github.com/tedrasiel)**  
_Wednesday Apr 01, 2015 at 17:16 GMT_

----

when i remember right, if youre not hitcapped u can "miss" against mobs too, but the second u are , u shouldnt miss on them (if they're on your lvl- range or lower). but no proof for that one

**Comment by netherspite** [comment]
<a href="https://github.com/jovce17"><img src="https://avatars3.githubusercontent.com/u/2951412?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [jovce17](https://github.com/jovce17)**  
_Wednesday Apr 01, 2015 at 18:18 GMT_

----

as far i can remember you need 18% to not miss on bosses 15 vs lvl 70 mobs and even less as the mobs level goes down as far i do remember you dont even need any hit rating vs lvl 66 mobs  >3 lvls bellow you ...

**Comment by netherspite** [comment]
<a href="https://github.com/nz54445"><img src="https://avatars1.githubusercontent.com/u/8640523?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [nz54445](https://github.com/nz54445)**  
_Wednesday Apr 01, 2015 at 19:30 GMT_

----

If you look at the link it's clear that it's 17%, and I am 100% sure of that, because draenai ele shamans only need 4% hit: 9% from talents, 3% from wrath and 1% from aura = 13% + 4% = 17%.

Furthermore you only need 6% on Heroics, because no mob is higher than 72.

Bottom line is, hit can be capped and you shouldn't ever miss mobs when you are hit capped, or it should happen very very rarely. There might still be a base miss chance but i can't find the link anywhere.

**Comment by netherspite** [comment]
<a href="https://github.com/ccshiro"><img src="https://avatars2.githubusercontent.com/u/8135895?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ccshiro](https://github.com/ccshiro)**  
_Wednesday Apr 01, 2015 at 19:52 GMT_

----

Stepped through with a debugger. The scenario you described is using a 1% chance to miss, which is the intended behavior. If you want to statistically analyze this (might be a good idea in case something later down the road goes wrong), I suggest a lot more trials.
